CondTools/SiPixel/test/createTestDBObjects.sh

```diff
@@ -11,10 +11,10 @@ echo "TESTING Pixel 1D GenErr DB code ..."
 cmsRun ${LOCAL_TEST_DIR}/SiPixelGenErrorDBObjectUploader_cfg.py MagField=0.0 Fullname=SiPixelGenErrorDBObject_phase1_0T_mc_BoR3_v1_bugfix Map=${LOCAL_TEST_DIR}/../data/phaseI_mapping.csv GenErrFilePath=CalibTracker/SiPixelESProducers/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1 || die "Failure running SiPixelGenErrorDBObjectUploader_cfg.py" $? 
 
 echo "TESTING Pixel 1D Template DB code for Phase-2 ..."
-cmsRun ${LOCAL_TEST_DIR}/SiPixelTemplateDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=7 Append=mc_25x100 Map=${LOCAL_TEST_DIR}/../data/Phase2_25by100_templates_2020October/phase2_T15_mapping.csv geometry=T15 TemplateFilePath=CondTools/SiPixel/data/Phase2_25by100_templates_2020October || die "Failure running SiPixelTemplateDBObjectUploader_Phase2_cfg.py" $? 
+cmsRun ${LOCAL_TEST_DIR}/SiPixelTemplateDBObjectUploader_Phase2_cfg.py MagField=3.8 Version=7 Append=mc_25x100 Map=$CMSSW_BASE/src/CondTools/SiPixel/data/Phase2_25by100_templates_2020October/phase2_T15_mapping.csv geometry=T15 TemplateFilePath=CondTools/SiPixel/data/Phase2_25by100_templates_2020October || die "Failure running SiPixelTemplateDBObjectUploader_Phase2_cfg.py" $? 
```

Comment:
I'm not sure CalibTracker-SiPixelESProducers and CondTools/SiPixel are the same thing. Would you try chaning this to CalibTracker/SiPixelESProducers becase your PR is actually for that package/subpackage